src/Components/Product/ProductCard.jsx

```diff
@@ -0,0 +1,51 @@
+import { AddToWishList } from "../Wishlist/AddToWishlist";
+
+import { AddToCart } from "../Cart/AddToCart";
+
+export function ProductCard({ product }) {
+  const {
+    brand,
+    categoryName,
+    price,
+    rating,
+    originalPrice,
+    img,
+    newArrival,
+    stockQuantity,
+  } = product;
+
+  return (
+    <div className="card">
+      <img className="card-img" src={img} alt="product" />
+      <p className="heading-3 l-sp-1 theme-shade-2 txt-bold">{brand}</p>
+      <span className="card-desc">{categoryName}</span>
+      <p className="card-price txt-bold">
+        ₹ {price}
+        <span className="txt-small txt-bold l-through txt-grey">
+          {" "}
+          ₹ {originalPrice}
+        </span>
+      </p>
+      <span
+        className="rating-block txt-white"
+        style={{ backgroundColor: rating >= 3 ? "#26a541" : "#ff6161" }}
```

Comment:
Liked this detailing of UX. 
Can you move this inline styling to class based styling?

Reply:
it's for specific use only , so not make a difference i